comps/imagegen/launch_tritonserver.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+
+default_port=8080
+default_card_num=0
+default_model_cache_directory="${HOME}/.cache/huggingface/hub"
+HUGGINGFACEHUB_API_TOKEN=${HUGGINGFACEHUB_API_TOKEN}
+
+docker_cmd=<<EOF
+docker run -d \
+    --name=TritonStabilityServer -p ${port_number}:8000 \
```

Comment:
Is the idea that this will fail cause of the set -u at the top? Or maybe this should be default_port?

Reply:
You're right - fixing. Thank you!